### ISSUE TITLE
Error with tscn with scene that has sprite as a root node

### ISSUE THREAD

**Issue by brainard52** [godotengine/godot#24600]
**Godot version:**

v3.0.6.stable.official.8314054 (installed with brew)

**OS/device including version:**

macOS 10.14.2 (18C54)

**Issue description:**

Error: "core/io/image_loader.cpp:55 - Error opening file: res://path/to/scene.tscn"
I'm not certain what more to put here. I've replicated this in multiple user accounts to ensure it wasn't some sort of cache or saved application state. I also tried reinstalling Godot. 

**Steps to reproduce:**
1. Create a scene
2. Add a sprite (As far as I can tell, it needs to be the root node)
3. Create a new AtlasTexture for Texture
4. Edit the new AtlasTexture (you do not need to select a texture)
5. Change region to have h of 48 and w of 128

**Minimal reproduction project:**

[spritetest.tar.gz](https://github.com/godotengine/godot/files/2709625/spritetest.tar.gz)

**Comment by Zylann** [comment]
It is strange because the `AtlasTexture` it complains about was successfully loaded.
```
ERROR: load_image: Error opening file: res://Test2.tscn::1
   At: core/io/image_loader.cpp:55
```

**Comment by KoBeWi** [comment]
So, is this important to fix for 3.0? It's just random error that doesn't affect anything.

**Comment by Calinou** [comment]
@brainard52 Have you been able to reproduce this in 3.1.1 or the `master` branch?

**Comment by KoBeWi** [comment]
In case there's no response, I can confirm that it's fixed in 3.1.

**Comment by akien-mga** [comment]
Closing as fixed in 3.1. In case someone wants to fix it in the 3.0.x branch, the commit that fixed it in 3.1 could likely be bisected and cherrypicked, but it's not a bad bug so not worth keeping an issue open about it.
